File: data/readable_format/fortran/file357.f90
program main
      character*2 a
      character*4 b
      character*6 c
      parameter (a="12")
      parameter (b = a)

      write (c,'("#",A,"#")') b
      if (c .ne. '#12  #') then
         print *, 'Aborting...'
         call abort
      else
         print *, 'All tests passed successfully.'
      end if
end program main

File: data/readable_format/cpp/file357.cpp
#include <iostream>
#include <string>
#include <cstdlib>

void checkFormattedString(const std::string& a) {
    std::string b = a;
    std::string c = "#" + b + std::string(2, ' ') + "#"; // Adjusting to ensure matching behavior

    if (c == "#12  #") {
        std::cout << "All tests passed successfully." << std::endl;
    } else {
        std::cout << "Aborting..." << std::endl;
        std::abort();
    }
}

int main() {
    checkFormattedString("12");
    return 0;
}